plugins/shortcuts/src/ShortcutItem.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { SidebarItem } from '@backstage/core';
+import { IconButton, makeStyles } from '@material-ui/core';
+import EditIcon from '@material-ui/icons/Edit';
+import { ShortcutIcon } from './ShortcutIcon';
+import { EditShortcut } from './EditShortcut';
+import { ShortcutApi } from './api';
+import { Shortcut } from './types';
+
+const useStyles = makeStyles({
+  icon: {
+    color: 'white',
+    fontSize: 16,
+  },
+});
+
+const getIconText = (title: string) =>
+  title.split(' ').length === 1
+    ? // If there's only one word, keep the first two characters
+      title[0].toUpperCase() + title[1].toLowerCase()
+    : // If there's more than one word, take the first character of the first two words
+      title
+        .replace(/\B\W/g, '')
+        .split(' ')
+        .map(s => s[0])
+        .join('')
+        .slice(0, 2)
+        .toUpperCase();
+
+type Props = {
+  shortcut: Shortcut;
+  api: ShortcutApi;
+};
+
+export const ShortcutItem = ({ shortcut, api }: Props) => {
+  const classes = useStyles();
+  const [displayEdit, setDisplayEdit] = useState(false);
```

Comment:
Ok, I _think_ I did it the correct way, please take a look.

Now that it's only using CSS I don't really know how this can be tested, seems testing for css properties is not that straight forward... I ended up removing a test for asserting the displaying of the edit button/icon all together. Do you think we need tests for this and if so do you have any pointers on where to start? @Rugvip